mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -1525,3 +1524,24 @@ func clusterConnectionStringsSchema() *schema.Schema {
 		},
 	}
 }
+
+func isEqualProviderAutoScalingMinInstanceSize(k, old, newStr string, d *schema.ResourceData) bool {
+	canScaleDown, _ := d.GetOk("auto_scaling_compute_scale_down_enabled")
+	canScaleUp, _ := d.GetOk("auto_scaling_compute_enabled")
+	if canScaleDown != nil && canScaleUp != nil && canScaleUp.(bool) && canScaleDown.(bool) {
```

Comment:
Nitpick: can we simplify this? kinda hard to read and yeah well it's a four conditions if

Reply:
I splitted into 2 blocks this condition